packages/eslint-plugin-gestalt/src/prefer-flex.test.js

```diff
@@ -0,0 +1,62 @@
+// @flow strict
+import rule, { errorMessage } from './prefer-flex.js';
+import {
+  getPathFormatterByRuleName,
+  getRuleTester,
+  getTestTypePrepender,
+  readTestByPath,
+} from './testHelpers.js';
+
+const ruleName = 'prefer-flex';
+const ruleTester = getRuleTester();
+const pathFormatter = getPathFormatterByRuleName(ruleName);
+
```

Comment:
for my last one, I added 

`const buildInvalidTest = (name) => readTestByPath(pathFormatter(invalidPrepender(name)));`